src/main/kotlin/no/nav/syfo/varsel/arbeidstaker/ArbeidstakerVarselApi.kt

```diff
@@ -73,13 +73,14 @@ fun Route.registerArbeidstakerVarselApi(
 
                 val hasAccessToVarsel = motedeltakerArbeidstaker.personIdent == requestPersonIdent
                 if (hasAccessToVarsel) {
-                    if (referat != null) {
+                    if (referat != null && referat.lestDatoArbeidstaker == null) {
                         dialogmotedeltakerService.lesReferatArbeidstaker(
                             personIdentNumber = requestPersonIdent,
                             dialogmotedeltakerArbeidstakerUuid = motedeltakerArbeidstaker.uuid,
                             referatUuid = varselUuid,
                         )
-                    } else {
+                    }
+                    if (motedeltakerArbeidstakerVarsel != null && motedeltakerArbeidstakerVarsel.lestDato == null) {
```

Comment:
Er bare max en av disse if'ene som vi slå til i alle fall, så trengs ingen else her.

Reply:
Da trenger man vel ikke to ifer?

Reply:
Presisering: max en av disse if'ene kan slå til, eller ingen av dem. Hvis ingen av dem slår til betyr det at referatet/varselet har blitt lest tidligere og da gjør vi ikke noe annet enn å returnere OK.